meta-spi/recipes-config/spi-config/spi-config_0.1.bb

```diff
@@ -0,0 +1,44 @@
+# See http://git.yoctoproject.org/cgit.cgi/poky/tree/meta/files/common-licenses
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
```

Comment:
@shreyan-collab you can't use MIT for this code, because it's GPL version 2.  See https://github.com/cu-ecen-aeld/Finalproject-Shreyan-Bala-shared/blob/master/spi_package/spidevtest.c#L1-L9

Reply:
@dwalkes You want me to change the license and push the code again?

Reply:
@dwalkes I have modified the license to GPL Version 2